packages/utils/platform-tools/src/Platform.ts

```diff
@@ -0,0 +1,9 @@
+import { Platform } from 'react-native';
+
+export const isMobile = Platform.OS === 'android' || Platform.OS === 'ios';
+export const isDesktop = Platform.OS === 'macos' || Platform.OS === 'windows' || Platform.OS === ('win32' as any);
+export const isAndroid = Platform.OS === 'android';
+export const isIOS = Platform.OS === 'ios';
+export const isMac = Platform.OS === 'macos';
+export const isWin32 = Platform.OS === ('win32' as any);
+export const isWindows = Platform.OS === 'windows';
```

Comment:
I was taught early in my career to avoid `utils` objects / classes / etc as they become a catch all. I can see the usefulness of a platform-tools style package. In its current state, I'm not sure these are all that useful. For example: `isMobile` includes iPad where we might want the desktop experience. So I've actually written that check with an extra `Platform.isPad` in it as well. But then iPad has different size classes if you take an app and slide it over another app. In that case, you want the mobile experience again. So then we had to write an extra (hacky) hook to guess that. I could also see this being useful to say, have a helper function that returns components as null if on an unsupported platform (I'm still thinking about that one, I haven't thought how that may best look). In its current form of just doing a bunch of single line if checks, I'm not a fan of it. 

I think I'd prefer the creation of a package like this in a separate PR that _just_ creates the package, and refactors platform checks across the repo to accommodate.

Reply:
Publishing a new package also tends to break our somewhat fragile nom pipeline for a bit. So extra reason to put it in its own PR.

Reply:
I have copied the changes locally for different branch and will raise a different PR for it.  
For now, I will have a local check in the file it itself.